docker-compose.yml

```diff
@@ -46,11 +46,14 @@ x-op-backend: &backend
     OPENPROJECT_WEB_MAX__THREADS: 1
     OPENPROJECT_WEB_MIN__THREADS: 1
     OPENPROJECT_WEB_WORKERS: 0
+    PIDFILE: /home/dev/openproject/tmpfs/pids/server.pid
   volumes:
     - ".:/home/dev/openproject"
     - "opdata:/var/openproject/assets"
     - "bundle:/usr/local/bundle"
     - "tmp:/home/dev/openproject/tmp"
+  tmpfs:
+    - /home/dev/openproject/tmpfs/pids:uid=$DEV_UID,gid=$DEV_GID
```

Comment:
❓ Do you have a reference to this syntax? I'm not sure, how compose should interpret everything after the colon `:`. I cannot find it in the [compose docs](https://docs.docker.com/compose/compose-file/05-services/#tmpfs).

Reply:
I don't have a pointer to official docs. However, I believe I picked it up from here https://github.com/docker/compose/issues/3425#issuecomment-423091370